.gitpod.yml

```diff
@@ -36,9 +36,10 @@ tasks:
     openMode: split-right
 vscode:
   extensions:
-    - hangxingliu.vscode-nginx-conf-hint@0.1.0:UATTe2sTFfCYWQ3jw4IRsw==
-    - zxh404.vscode-proto3@0.4.2:ZnPmyF/Pb8AIWeCqc83gPw==
-    - ms-kubernetes-tools.vscode-kubernetes-tools@1.2.1:j58HdmA0K7d9a9sEkogZNw==
-    - bajdzis.vscode-database@2.2.1:uXdjV53wtoTevFK6HOh3pQ==
-    - hashicorp.terraform@2.1.1:QEP7gdWtMuY+j8RZ5OLDkA==
-    - stkb.rewrap@1.13.0:yEx8nRSXlfXAHqsbkTJpdA==
+    - golang.go
```

Comment:
Should I revert this section after PR is approved?

Reply:
Did you update it for testing, or was there another reason?

Reply:
for testing, but we can also stick to them to feel the pain first 😄

Reply:
if it's just for testing, please revert. otherwise, if it's necessary, it's good to leave it, no?

Reply:
golang.go will be necessary for Code, since we don't want to have it as built-in there, but rest will revert